application/displayblock.class.inc.php

```diff
@@ -1009,14 +1009,18 @@ public function GetRenderContent(WebPage $oPage, array $aExtraParams, $sId)
 				$iTotalCount = 0;
 				$aValues = array();
 				$aURLs = array();
+				$iMaxNbCharInLabel = 0;
 				foreach ($aRes as $iRow => $aRow)
 				{
 					$sValue = $aRow['grouped_by_1'];
 					$sHtmlValue = $oGroupByExp->MakeValueLabel($this->m_oFilter, $sValue, $sValue);
 					$aGroupBy[(int)$iRow] = (int) $aRow[$sFctVar];
 					$iTotalCount += $aRow['_itop_count_'];
 					$aValues[] = array('label' => html_entity_decode(strip_tags($sHtmlValue), ENT_QUOTES, 'UTF-8'), 'label_html' => $sHtmlValue, 'value' => (int) $aRow[$sFctVar]);
-					
+					if ($iMaxNbCharInLabel < strlen($sValue)) {
+						$iMaxNbCharInLabel = strlen($sValue);
```

Comment:
```suggestion
					if ($iMaxNbCharInLabel < strlen($sValue)) {
						$iMaxNbCharInLabel = strlen($sValue);
```

Rather use [mb_strlen](https://www.php.net/manual/fr/function.mb-strlen.php) to ensure it works as expected with non latin alphabets.